### PR TITLE
feat:change the modal with js functionality

### PR DESCRIPTION
**Pull request by mariyasada**
added one button and after click, this button modal went open

https://user-images.githubusercontent.com/77275100/159734442-739e9aa5-0fe8-41be-8e0b-4d04978849be.mp4

### CODE REVIEW

File: stylesheets/modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+modalCodeBtn.addEventListener("click", () => {
+        console.log("clicked");
+        modalCodeDiv.style.display == "block"
+          ? (modalCodeDiv.style.display = "none")
+          : (modalCodeDiv.style.display = "block");
+      });
+
+// const modalCodeBtn = document.querySelector("#modal-code-btn");
```
Comment (shaksham08) [inline stylesheets/modal/modal.js]: Here you can remove the commented code

File: stylesheets/documentation.css [hunk 2]
```diff
@@ -15,20 +15,20 @@
...
-    flex: 1;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 80vw;
+  padding: 0.6rem;
+  gap: 2.5rem;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  /* flex: 1; */
```
Comment (shaksham08) [inline stylesheets/documentation.css:26]: commented code?

**Review by shaksham08 — APPROVED** [review]
Code looks clean and good just added comments you can refer

File: stylesheets/modal/modal.js (hunk 1 above)
Comment (mariyasada) [reply stylesheets/modal/modal.js]: done

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
This PR is good to merge.

**Pull request merged**